React/Views/RCTTVView.m

```diff
@@ -465,4 +482,20 @@ - (void)addFocusGuide:(NSArray*)destinations {
   self.focusGuide.preferredFocusEnvironments = destinations;
 }
 
+- (void)setTrapFocusUp:(BOOL)enabled {
+  self->_trapFocusUp = enabled;
```

Comment:
Not sure you really need these setters, these are what get synthesized by the "assign" property declarations.

Reply:
Yeah, you're absolutely right. They're totally unnecessary 😄 Removed ✅